docs/en/ingest-management/elastic-agent/configuration/providers/elastic-agent-providers.asciidoc

```diff
@@ -89,19 +91,19 @@ configuration.
 * <<kubernetes-provider,Kubernetes Provider>>
 
 [discrete]
-[[disable-providers-by-default]]
-=== Disabling Providers By Default
+[[enable-providers-by-default]]
+=== Enabling providers By default
 
-All registered providers are enabled by default.
+All registered providers are disabled by default.
 
-Disable all providers by default and only enable explicitly configured providers by setting `agent.providers.initial_default: false`.
+Enable all providers by default and only disable explicitly configured providers by setting `agent.providers.initial_default: true`.
```

Comment:
You should really keep this section how it was. As this still applies.

If you set `initial_default: false` and you have a provider referenced in the policy the provider still will not run.